samples/CMakeLists.txt

```diff
@@ -33,9 +33,6 @@ endif()
 if((NOT ANDROID) AND HAVE_OPENGL)
   add_subdirectory(opengl)
 endif()
-if(HAVE_OPENVX)
-  add_subdirectory(openvx)
```

Comment:
Sanity check still shows OpenVX reference on the line 131:

```
$ grep -Rni openvx ./
```

Reply:
addressed👍🏽